deploy/klusterlet/olm-catalog/klusterlet/manifests/klusterlet.clusterserviceversion.yaml

```diff
@@ -209,6 +209,13 @@ spec:
           verbs:
           - update
           - patch
+        - apiGroups:
+          - work.open-cluster-management.io
```

Comment:
need update rbac in config dir firstly, and then update csv by `make update-csv`